package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "vanilla-framework",
-  "version": "4.12.1",
+  "version": "4.12.2",
```

We haven't released 4.12.1 yet, so no further changes to version number are needed (unless we need to bump it to 4.13 minor one)

```suggestion
  "version": "4.12.1",
```